tests/unit/test_releases.py

```
@@ -0,0 +1,130 @@
+        assert created.id is not None
+        assert created.name == name
+
+        try:
+            updated = client.releases.update(
+                workspace_slug,
+                created.id,
+                UpdateRelease(description="Updated release description"),
+            )
+            assert updated.id == created.id
+            assert updated.description == "Updated release description"
+        except Exception as exc:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)